server/events/approve_policies_command_runner.go

```diff
@@ -9,7 +9,7 @@ import (
 func NewApprovePoliciesCommandRunner(
 	commitStatusUpdater CommitStatusUpdater,
 	prjCommandBuilder ProjectApprovePoliciesCommandBuilder,
-	prjCommandRunner ProjectPolicyCheckCommandRunner,
+	prjCommandRunner ProjectApprovePoliciesCommandRunner,
```

Comment:
Adding this change here is going to make it tricky to upstream this as the refactor only exists in our fork.   Maybe I should just upstream the refactor

Reply:
It should be trivial to fix it upstream. I will update it, also your refactor builds on top of my policy_check PR that had some refactoring already done. We can potentially include your refactor in my PR.